src/index.ts

```diff
@@ -76,14 +77,10 @@ export class LinkChecker extends EventEmitter {
     return result;
   }
 
-  /**
-   * Spin up a local HTTP server to serve static requests from disk
-   * @param root The local path that should be mounted as a static web server
-   * @param port The port on which to start the local web server
-   * @private
-   * @returns Promise that resolves with the instance of the HTTP server
-   */
-  private startWebServer(root: string, port: number): Promise<http.Server> {
+  private _startStaticWebServer(
```

Comment:
Yeah let's add the jsdoc back

Reply:
Yeah I should have broken this up into 3 commits at least since they are orthogonal.

1. prettier.
2. crawl function split
3. js doc comments removal

Will do that at some point, as well as the interesting parallelization bit (if someone else doesn't beat me to it first). Great feedback, thanks!

Reply:
added comments back